pkg/operator/workloadcontroller/apigroup.go

```diff
@@ -31,9 +32,13 @@ func checkForAPIs(restclient rest.Interface, groupVersions ...schema.GroupVersio
 		url := "/apis/" + groupVersion.Group + "/" + groupVersion.Version
 
 		statusCode := 0
-		restclient.Get().AbsPath(url).Do().StatusCode(&statusCode)
+		result := restclient.Get().AbsPath(url).Do().StatusCode(&statusCode)
+		resultBody, err := result.Raw()
```

Comment:
handle err better. It's an error case of the request, don't mix it up with a http error code.